util/clock/time_test.go

```diff
@@ -53,8 +57,12 @@ func TestSleepContext(t *testing.T) {
 }
 
 func TestSleepSource(t *testing.T) {
+	t.Parallel()
+
 	for _, tc := range cases {
 		t.Run(fmt.Sprintf("%v:%v", tc.dur, tc.timeout), func(t *testing.T) {
+			t.Parallel()
```

Comment:
I think this will fall foul of https://github.com/golang/go/wiki/CommonMistakes#using-goroutines-on-loop-iterator-variables.